tests/unit/collapsible/collapsible_core.js

```diff
@@ -1,6 +1,11 @@
 test( "Pre-rendered nested collapsibles are enhanced correctly", function() {
-var outerWidget = $( "#outer" ).collapsible().data( "mobile-collapsible" ),
-	innerWidget = $( "#inner" ).collapsible().data( "mobile-collapsible" );
+
+//This Test previously was actually testing auto page enhancment unintentionally
+// we use enhance to not worry about each widget
```

Comment:
Please write "we" with an uppercase "W", because this is another sentence ... or just feel free to make both sentences into one sentence that wraps at the 100 character limit :)